documentation/en/source/pages/templates-spec/stack-bundles.rst

```diff
@@ -205,10 +205,20 @@ To get the list of categories available, run command:
 
 	$ hammr bundle categories
 
-Adding oses
-~~~~~~~~~~~
+Adding restriction rule
+~~~~~~~~~~~~~~~~~~~~~~~
 
-The following example show how you can provide oses for which the bundle is designed.
+The following example show how you can provide restrictions.
+
+The restriction rule is represented by a "logical" expression.
+A simple term must match pattern "Object#field=value" or "Object#field!=value".
+   - object is "Distribution" or "TargetFormat"
+   - field is "family, pkgType, name, version or arch" for Distribution object (see :ref:`command-line-os` section and os list command to find values for field)
+   - field is "name or type" for TargetFormat object (see :ref:`template-builders` section to get values for name field. Values for type field are cloud, virtual, physical or container)
```

Comment:
field is ``name`` or ``type`` for TargetFormat object (refer to :ref:`template-builders` section to get values for ``name`` field. Values for ``type`` field are cloud, virtual, physical or container)